bootstrap.sh

```diff
@@ -4,9 +4,6 @@ set -eu
 ## Make sure we have Ruby 3 installed directly on the instance
 amazon-linux-extras install -y ruby3.0
 
-## Use Docker Buildkit instead of the legacy builder: https://docs.docker.com/build/buildkit/
-export DOCKER_BUILDKIT=1
```

Comment:
It turns out this is not the correct space for this. Removing it since it does nothing and has been added to the `panorama-buildkite-secrets` s3 bucket `env` object to correctly be loaded into the agent env